### PR TITLE
Add RabbitMQ as recommended system-requirement

### PR DESCRIPTION
**Pull request by skoch98**
As discussed with Bernhard Rusch at the Inspire, I am now adding RabbitMQ as a recommended system requirement in the documentation. I tried to present RabbitMQ's arguments as convincingly as possible... 😄

### CODE REVIEW

**Comment by brusch** [comment]
LGTM, I think we should also include an example of how to configure RabbitMQ, that would make it more convenient for users 😊

**Comment by skoch-twocream** [comment]
> LGTM, I think we should also include an example of how to configure RabbitMQ, that would make it more convenient for users 😊

I would use this scheme as an example:
 - https://github.com/pimcore/skeleton/blob/fbd28ac2b65fe81bb69b581c94191e80bce18a88/.docker/messenger.yaml

This way there is an .ENV varaible for the base DSN and it is possible to use multiple queues.

Alternatively, I can document it in a simplified way so that the .ENV contains the queue directly and therefore all tasks run into it (I find it not recommend but possibly easier and more error-free to understand for the usual user).

**Comment by brusch** [comment]
For me it would be fine that we just reference to the skeleton file. 
So I'd then finish https://github.com/pimcore/skeleton/pull/187 first and then update this PR.

**Comment by brusch** [comment]
Btw. just released v3.2 of our docker image with AMQP support ... just building at the moment: 
https://github.com/pimcore/docker/actions/runs/8520253593

**Review by fashxp — COMMENTED** [review]
can we also make that statement consistent: https://github.com/pimcore/pimcore/blob/11.x/doc/01_Getting_Started/02_Advanced_Installation_Topics/01_Symfony_Messenger.md?plain=1#L22

**Comment by skoch98** [comment]
> can we also make that statement consistent: https://github.com/pimcore/pimcore/blob/11.x/doc/01_Getting_Started/02_Advanced_Installation_Topics/01_Symfony_Messenger.md?plain=1#L22

For `pimcore_core` sure. But personally, I would leave the failed queue in the database so that it is easier to view it and report it if there are errors. What is your vendor view on this?

**Comment by brusch** [comment]
@skoch98 yes, I'd leave failed queue in db as well 👍

**Comment by skoch98** [comment]
I updated the docs but please make sure that https://github.com/pimcore/skeleton/pull/187 will merged first so that the link is valid :-)

Edit: I think this is better for everyone: 48030446a8be1d1fc9c191cf2d76fd1c02c605ad

File: doc/01_Getting_Started/02_Advanced_Installation_Topics/01_Symfony_Messenger.md [hunk 1]
```diff
@@ -13,10 +13,12 @@ framework:
 
             pimcore_core:
                 dsn: "doctrine://default?queue_name=pimcore_core"
+                # For RabbitMQ (recommend) use this as example:
```
Comment (jdreesen) [inline doc/01_Getting_Started/02_Advanced_Installation_Topics/01_Symfony_Messenger.md]:
```suggestion
                # For RabbitMQ (recommended) use this as example:

```

**Comment by mcop1** [comment]
Since https://github.com/pimcore/docker/pull/152 was merged, the image was released and the pr looks good, I merged it.
Thanks everyone for creating/improving the pr :+1:

**Pull request merged**